pages/index.tsx

```diff
@@ -32,39 +32,14 @@ class Index extends React.Component<IndexProps & WithPageMetadataProps> {
       <Page pageMetadata={this.props.pageMetadata} isHome={true} title="Home">
         <div className="container">
           <Alert kind="info">
-            <h2>Website Statement</h2>
+            <p>The DDD Perth Committee has regretfully postponed the 2019 conference to 2020.</p>
```

Comment:
`postponed the 2019 conference to 2020`? Is this correct? Should It be `postponed the 2020 conference to 2021` ?

Reply:
oooooooh, good catch Mo! Will fix that up.